tensorflow/lite/micro/benchmarks/Makefile.inc

```diff
@@ -1,9 +1,9 @@
 KEYWORD_BENCHMARK_SRCS := \
-tensorflow/lite/micro/benchmarks/keyword_benchmark.cc \
-tensorflow/lite/micro/benchmarks/keyword_scrambled_model_data.cc
```

Comment:
it looks like with this line removed, keyword_scrambled_model_data.cc is not listed anywhere. So I'm a bit confused why the benchmark would build with this change.

Reply:
The main Makefile, line 365 adds the model data files to MICROLITE_CC_BASE_SRCS.

Another possible fix would be to add the model data files to MICROLITE_BENCHMARK_SRCS since the files there are filtered out from MICROLITE_CC_SRCS. 

If you prefer that option I'll upload a fix tomorrow.

Reply:
That would be nicer, but will likely break the memory_arena_threshold_test added with https://github.com/tensorflow/tensorflow/commit/d8881eb71d3bb4e1a8a2358cad848b11d664b675.

I'll try and see if I can come up with a fix that I am happy with tomorrow.